Standards/CCSMB-1.md

```diff
@@ -0,0 +1,14 @@
+# CCSMB-1: Procedures for submitting a standard
```

Comment:
This should be capitalized according to standard English APA/Whatever rules
https://capitalizemytitle.com/#capitalizationrules

Reply:
Maybe reword it a bit too:
```suggestion
# CCSMB-1: Procedures for Standards Submission
```